example/alertmanager-service.yaml

```diff
@@ -2,6 +2,9 @@ apiVersion: v1
 kind: Service
 metadata:
   name: alertmanager-main
+  labels:
+    app: alertmanager
+    instance: main
```

Comment:
I think in our stateful sets etc. we went with `app: alertmanager` and `alertmanager: main`.
`instance` is only really meaningful in combination of both labels whereas the other one can serve as a meaningful selector in itself.